io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -172,19 +236,48 @@ def cc_operations(self):
         """
         for state in ['start', 'stop', 'start', 'pause', 'resume']:
             self.cc_state(state)
-            self.cc_showprogress()
-        while self.cc_showprogress() is True:
+            cmd = "./storcli64 /c%d/v0 show cc" % self.controller
+            self.showprogress(cmd)
+        while self.showprogress(cmd) is True:
             time.sleep(30)
 
     def cc_state(self, state):
 
         """
         Helper function for all CC operations
         """
-        cmd = "./storcli64 /c%d/v0 %s cc force" % (self.controller, state)
+        if state == 'start':
+            cmd = "./storcli64 /c%d/v0 %s cc force" % (self.controller, state)
+        else:
+            cmd = "./storcli64 /c%d/v0 %s cc" % (self.controller, state)
         self.check_pass(cmd, "Failed to %s CC" % state)
         time.sleep(10)
 
+    def full_init(self):
+
+        """
+        Helper function to start Fast/Full init
+        """
+        cmd = "./storcli64 /c%d/vall start init full" % self.controller
+        self.check_pass(cmd, "Failed to start init")
+        cmd = "./storcli64 /c%d/vall show init" % self.controller
+        self.showprogress(cmd)
+        while self.showprogress(cmd) is True:
+            time.sleep(30)
```

Comment:
@balamuruhans  @narasimhan-v  @PraveenPenguin  Thanks for all your comments. I have changed accordingly where ever needed. But I felt its better to have 30sec as sleep time, so didnt made any changes there.